Chaincase.UI/Chaincase.UI.csproj

```diff
@@ -32,11 +32,21 @@
     <ItemGroup>
       <Folder Include="ViewModels\" />
       <Folder Include="Components\" />
+      <Folder Include="wwwroot\style\" />
     </ItemGroup>
 
     <ItemGroup>
       <Compile Remove="ViewModels\SendAmountPage.xaml.cs" />
     </ItemGroup>
+    <ItemGroup>
+      <Content Remove="wwwroot\style\EmptyCSSFile.css" />
+    </ItemGroup>
+    <ItemGroup>
+      <EmbeddedResource Include="wwwroot\style\global.css">
+        <ExcludeFromSingleFile>true</ExcludeFromSingleFile>
+        <CopyToPublishDirectory>PreserveNewest</CopyToPublishDirectory>
+      </EmbeddedResource>
+    </ItemGroup>
```

Comment:
I think both ItemGroups with Remove can be removed. global.css should be included as Content rather than Embedded resource too I believe. buuuuut everything else looks good !